src/SmartComponents/ActivityTable/__tests__/ActivityTable.tests.js

```diff
@@ -36,7 +36,7 @@ describe('ActivityTable', () => {
     jest.clearAllMocks();
   });
 
-  it('should render correctly', () => {
+  it.skip('should render correctly', () => {
```

Comment:
Why do all these need to be skipped?

Reply:
I didn't mean for them all to be skipped. I forgot to change them. Good catch.